api/src/__tests__/cohort.test.ts

```diff
@@ -16,147 +16,228 @@ jest.mock("../emails", () => ({
 beforeAll(dbConnect);
 afterAll(dbClose);
 
-describe("PUT /:cohort", () => {
-  it("should return 400 if cohort name is invalid", async () => {
-    const res = await request(getAppHelper()).put("/cohort/NonValidName").send({ name: "New Cohort" });
-    expect(res.status).toBe(400);
+describe("Cohort", () => {
+  // @ts-ignore
+  passport.user.role = ROLES.ADMIN;
+  // @ts-ignore
+  passport.user.subRole = "god";
```

Comment:
```suggestion
  // @ts-ignore
  passport.user.role = ROLES.ADMIN;
  // @ts-ignore
  passport.user.subRole = "god";
```
on peux maintenant faire ça :
```js
request(getAppHelper({ role: ROLES.ADMIN, subRole: "god"))
```
le seul inconvénient c'est que dans le test suivant il faudra remettre `request(getAppHelper({ role: ROLES.ADMIN, subRole: undefined))` pour reset